### PR TITLE
Added a check to see if a query string is included or not, which proc…

### PR DESCRIPTION
**Pull request by jrauschenberg**
#### What's this PR do?

It includes a check in the WhenIWork.prototype.get function to see if a query string is already included (i.e. if there's already a question mark). In that case, it doesn't add a duplicate question mark. If not, it adds a question mark and then adds anything in the query object after the question mark, as it did before.

#### Where should the reviewer start? 

Look at lines 87 and 100 in index.js--those are the only that have changed.

#### How should this be manually tested? 

Call WhenIWork.prototype.get with a few sample sets of parameters, namely:
1. One set of parameters where method (params[0]) has a question mark and parameters after it already (e.g. 'shifts?include_objects=false').
2. One set of parameters where method has no question mark (e.g. 'shifts').
3. Have WhenIWork.prototype._makeRequest console.log the response it gets.

Make sure WhenIWork.prototype.get makes the appropriate request and gets the appropriate response.

#### What are the relevant tickets?

[INT-337](https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=5&view=detail&selectedIssue=INT-337)

#### Questions:

…eeds accordingly.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -96,7 +97,7 @@ WhenIWork.prototype.get = function () {
         u = u + i + '=' + requestParams[i] + '&';
```
Comment (apackin) [inline index.js]: Do we care that the query string ends with a '&'?

File: index.js [hunk 2]
```diff
@@ -96,7 +97,7 @@ WhenIWork.prototype.get = function () {
...
     }
 
-    var uri = method + '?' + u;
+    method.indexOf('?') === -1 ? uri = method + '?' + u : uri = method + '&' + u;
         
     this._makeRequest(uri, 'get', requestParams, headers, callback);
```
Comment (apackin) [inline index.js]: If we're already digesting the requestParams into the q string here, what does _makeRequest use them for?

**Comment by apackin** [comment]
This solution makes sense and works. I think it brings up some questions about the how _makeRequests should function differently for distinct methods that we may be interested in.

File: index.js (hunk 1 above)
Comment (jrauschenberg) [inline index.js]: I've tested and it makes no difference to WiW.

John

Comment (tongxiang) [inline index.js]: http://stackoverflow.com/questions/6663362/is-a-trailing-ampersand-legal-in-a-url

Comment (tongxiang) [inline index.js]: no problems with a trailing ampersand.

**Comment by tongxiang** [comment]
We need to update the version number here in order to publish. 

http://stackoverflow.com/questions/15056377/how-do-i-update-an-npm-module-that-i-published

@jrauschenberg Go ahead update the version number, push changes up here, and then we're good to merge. Then we should run `npm publish` to publish the new version.

**Pull request merged**